custom_components/plugwise/config_flow.py

```diff
@@ -77,7 +77,7 @@ def plugwise_stick_entries(hass):
     return sticks
 
 
-async def validate_usb_connection(self, device_path=None) -> dict[str, str]:
+async def validate_usb_connection(self, device_path=None) -> tuple[dict[str, str], Any]:
```

Comment:
@bouwew do you concur with this?

Reply:
If we want to make it more specific it should be: `tuple[dict[str, str], Stick | None]` I think.

Reply:
Tried that, but if we do that it throws the below on L206 and L230 on `await self.async_set_unique_id(api_stick.mac)` with:

```
homeassistant/components/plugwise/config_flow.py:206: error: Item "None" of "Optional[Any]" has no attribute "mac"  [union-attr]
homeassistant/components/plugwise/config_flow.py:230: error: Item "None" of "Optional[Any]" has no attribute "mac"  [union-attr]
```

But I'll flag it as a todo for now

Reply:
Ok, agree.